3rdparty/SConscript

```diff
@@ -306,6 +307,27 @@ elif 'sox' in system_dependencies:
 
     env = conf.Finish()
 
+# dep: openssl
+if 'openssl' in autobuild_dependencies:
+    env.BuildThirdParty(thirdparty_build_dir, meta.toolchain,
+        meta.thirdparty_variant, thirdparty_versions, 'openssl')
+elif 'openssl' in system_dependencies:
+    ossl_prefix = None
+    # macOS: fallback to use a brew-installed openssl (via PKG_CONFIG_PATH)
+    if meta.platform == 'darwin' and not is_crosscompiling:
+        ossl_prefix = env.FindBrewPackage('openssl')
+
+    conf = Configure(env, custom_tests=env.CustomTests)
+
+    if not conf.AddPkgConfigDependency('libssl', '--cflags --libs', prefix=ossl_prefix):
+        conf.env.AddPkgConfigLibs(['libssl'])
```

Comment:
nit: I suggest to rename `prefix` arg to something like `append_prefix` or `additional_prefix`, to make it clear that this only adds prefix to the search path, not replaces path.

Reply:
done